src/components/FeatureViews/AdminPortal/UnpaidSessionCard.js

```diff
@@ -27,41 +30,58 @@ const UnpaidSessionCard = ({unpaidStudent}) => {
 		adminUtils.calculateSessionLength(startTime, endTime)
```

Comment:
This produces a NaN amount due for some times because you parse the course times on lines 25-26, but `calculateSessionLength` also parses the time, causing NaN to be returned here and setting `amtDue` to NaN. Remove one of the parsing calls.